core/chains/terra/terratxm/txm.go

```diff
@@ -126,8 +132,33 @@ func (txm *Txm) run() {
 	}
 }
 
+var (
+	typeUrlSend    = sdk.MsgTypeURL(&types.MsgSend{})
```

Comment:
hm cleaner with just ```proto.MessageName(&types.MsgSend)``` to drop the prefix slash?

Reply:
then we could drop this URL suffix which is a bit confusing

Reply:
I don't fully understand it either, but it seems to be a pervasive standard throughout cosmos.

Reply:
Dropped the URL suffix and cleaned up naming a bit.